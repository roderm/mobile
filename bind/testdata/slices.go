package slices

type ByteArray []byte

type S struct {
	B []byte
	b []byte
}

type (s *S) Set(b1 []byte, b2 []byte) {
	s.b = b1
	s.B = b2
}

type (s *S) Get() ([]byte, []byte){
	return s.b, s.B
}