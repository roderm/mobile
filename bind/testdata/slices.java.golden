// Code generated by gobind. DO NOT EDIT.

// Java class slices.S is a proxy for talking to a Go program.
//
//   autogenerated by gobind -lang=java slices
package slices;

import go.Seq;